.github/workflows/authors.yml

```diff
@@ -0,0 +1,57 @@
+name: New Author Check
+
+on:
+  - pull_request
+
+jobs:
+  check-id:
+    name: Check Author ID
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt -y install git
+      - uses: actions/checkout@v3.2.0
+        with:
+          fetch-depth: 0
+      - name: Export all previous authors
+        run: git log --format="%an <%ae>" origin/${GITHUB_BASE_REF} | sort | uniq > authors.txt
+      - run: cat authors.txt
+      - name: Get authors of new commits
+        run: git log --format="%an <%ae>" origin/${GITHUB_BASE_REF}... | sort | uniq > commit-authors.txt
+      - run: cat commit-authors.txt
+      - name: Check new authors
+        run: |
+          while read -r author; do
+             echo "Checking author: ${author}"
+             if ! grep -q "^${author}\$" authors.txt; then
+                 echo "ERROR: ${author} NOT FOUND"
+                 echo "::error ::New author found: ${author}"
+                 echo "${author}" >> new-authors.txt
+                 echo has_new_authors="yes" >> $GITHUB_ENV
+             fi
+          done < commit-authors.txt
+      - if: ${{ env.has_new_authors == 'yes' }}
+        run: cat new-authors.txt
+      - if: ${{ env.has_new_authors == 'yes' }}
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          retries: 3
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'This pull request contains *NEW* authors.'
+            })
+      - if: ${{ env.has_new_authors != 'yes' }}
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          retries: 3
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'This pull request contains NO new authors.'
+            })
```

Comment:
Thank you for this! For testing this indeed looks good but wondering if it'd be noisy to post that there are no new authors. wdyt?